unittests/AST/StructuralEquivalenceTest.cpp

```diff
@@ -300,8 +300,7 @@ TEST_F(StructuralEquivalenceFunctionTest, ThrowVsNoexceptTrue) {
   EXPECT_FALSE(testStructuralMatch(t));
 }
 
-TEST_F(StructuralEquivalenceFunctionTest, DISABLED_NoexceptNonMatch) {
-  // The expression is not checked yet.
+TEST_F(StructuralEquivalenceFunctionTest, NoexceptNonMatch) {
```

Comment:
The expression itself is still not checked:
```
static bool IsStructurallyEquivalent(StructuralEquivalenceContext &Context,
                                     const Expr *E1, const Expr *E2) {
  if (!E1 || !E2)
    return E1 == E2;

  // FIXME: Actually perform a structural comparison!
  return true;
}
```
(But it is correct to assume a result of `true`.)